src/cloud-code/Parse.Cloud.js

```diff
@@ -8,40 +8,205 @@ function getClassName(parseClass) {
+ *
+ * ```
+ * Parse.Cloud.afterSave('MyCustomClass', async function(request) {
+ *   // code here
+ * })
+ *
+ * Parse.Cloud.afterSave(Parse.User, async function(request) {
+ *   // code here
+ * })
+ * ```
+ *
+ * @method afterSave
+ * @name Parse.Cloud.afterSave
+ * @param {(String|Parse.Object)} arg1 The Parse.Object subclass to register the after save function for. This can instead be a String that is the className of the subclass.
+ * @param {Function} func The function to run after a save. This function can be an async function and should take just one parameter, {@link Parse.Cloud.TriggerRequest}.
+ */
 ParseCloud.afterSave = function(parseClass, handler) {
   var className = getClassName(parseClass);
   triggers.addTrigger(triggers.Types.afterSave, className, handler, Parse.applicationId);
 };
 
+/**
+ * Registers an after delete function.
+ *
+ * **Available in Cloud Code only.**
+ *
+ * If you want to use afterDelete for a predefined class in the Parse JavaScript SDK (e.g. {@link Parse.User}), you should pass the class itself and not the String for arg1.
+ * ```
+ * Parse.Cloud.afterDelete('MyCustomClass', async (request) => {
+ *   // code here
+ * })
+ *
+ * Parse.Cloud.afterDelete(Parse.User, async (request) => {
+ *   // code here
+ * })
+ *```
+ *
+ * @method afterDelete
+ * @name Parse.Cloud.afterDelete
+ * @param {(String|Parse.Object)} arg1 The Parse.Object subclass to register the after delete function for. This can instead be a String that is the className of the subclass.
+ * @param {Function} func The function to run after a delete. This function can be async and should take just one parameter, {@link Parse.Cloud.TriggerRequest}.
+ */
 ParseCloud.afterDelete = function(parseClass, handler) {
   var className = getClassName(parseClass);
   triggers.addTrigger(triggers.Types.afterDelete, className, handler, Parse.applicationId);
 };
 
+/**
+ * Registers a before find function.
+ *
+ * **Available in Cloud Code only.**
+ *
+ * If you want to use beforeFind for a predefined class in the Parse JavaScript SDK (e.g. {@link Parse.User}), you should pass the class itself and not the String for arg1.
+ * ```
+ * Parse.Cloud.beforeFind('MyCustomClass', async (request) => {
+ *   // code here
+ * })
+ *
+ * Parse.Cloud.beforeFind(Parse.User, async (request) => {
+ *   // code here
+ * })
+ *```
+ *
+ * @method beforeFind
+ * @name Parse.Cloud.beforeFind
+ * @param {(String|Parse.Object)} arg1 The Parse.Object subclass to register the before find function for. This can instead be a String that is the className of the subclass.
+ * @param {Function} func The function to run before a find. This function can be async and should take just one parameter, {@link Parse.Cloud.BeforeFindRequest}.
+ */
 ParseCloud.beforeFind = function(parseClass, handler) {
   var className = getClassName(parseClass);
   triggers.addTrigger(triggers.Types.beforeFind, className, handler, Parse.applicationId);
 };
 
+/**
+ * Registers a after find function.
```

Comment:
`an after find` not `a after find`   (doh!)

Reply:
yeah I need to reword.